EMS/core-bundle/src/Resources/views/user/permissions/permissions.html.twig

```diff
@@ -0,0 +1,138 @@
+{% extends '@EMSCore/base.html.twig' %}{% trans_default_domain 'EMSCoreBundle' %}
+
+{% block pagetitle %}
+    {{ 'view.user.permissions.page_title'|trans }}
+{% endblock %}
+
+{% block body %}
+    <div class="row">
+        <div class="col-md-12">
+            <div class="box ">
+                <div class="box-header with-border">
+                    <i class="fa fa-sitemap"></i>
+                    <h3 class="box-title">{{ 'view.user.permissions.titles.content_type_permissions'|trans }}</h3>
+                </div>
+                <div class="box-body">
+                    <div class="table-responsive">
+                        <table class="table table-condensed table-striped">
+                            <tbody>
+                            <tr>
+                                <th class="text-center" style="width: 10px">{{ 'view.user.permissions.table_headers.content_type'|trans }}</th>
+                                <th class="text-center" style="width: 20px">{{ 'view.user.permissions.table_headers.icon'|trans }}</th>
+                                <th>{{ 'view.user.permissions.table_headers.view'|trans }}</th>
+                                <th>{{ 'view.user.permissions.table_headers.create'|trans }}</th>
+                                <th>{{ 'view.user.permissions.table_headers.edit'|trans }}</th>
+                                <th>{{ 'view.user.permissions.table_headers.publish'|trans }}</th>
+                                <th>{{ 'view.user.permissions.table_headers.delete'|trans }}</th>
+                                <th>{{ 'view.user.permissions.table_headers.trash'|trans }}</th>
+                                <th>{{ 'view.user.permissions.table_headers.archive'|trans }}</th>
+                                <th>{{ 'view.user.permissions.table_headers.show_link_create'|trans }}</th>
+                                <th>{{ 'view.user.permissions.table_headers.show_link_search'|trans }}</th>
+                                <th>{{ 'view.user.permissions.table_headers.specific_roles'|trans }}</th>
+                            </tr>
+                            {% for contentType in emsco_get_content_types() %}
+                                <tr class="text-{{ contentType.color }}">
+                                    <td class="text-center">{{ contentType.name }}</td>
+                                    <td class="text-center"><i class="{% if contentType.icon %}{{ contentType.icon }}{% else %}fa fa-book{% endif %}"></i></td>
+                                    {% for role in [contentType.roles.view, contentType.roles.create, contentType.roles.edit, contentType.roles.publish, contentType.roles.delete, contentType.roles.trash, contentType.roles.archive, contentType.roles.show_link_create, contentType.roles.show_link_search] %}
```

Comment:
The array of roles might be defined in the controller? And reused in the  @EMSCore/elements/get-button.html.twig template?